### PR TITLE
Update experiment-rollout.js

### PR DESCRIPTION
**Pull request by Lordcobcob**
Comitting  Changes to the branch adding a new rollout (Thats code may not work) and fixing some general grammar mistakes/clarifying strings.

### CODE REVIEW

File: experiment-rollout.js [hunk 1]
```diff
@@ -101,6 +101,26 @@ const data = {
...
+            },
+            {
+                type: 3,
+                value: [2, 50],
+                rate: 50,
+                ranges: [[1, 50]]
+            },
+        ]
+    },
+
```
Comment (Tolga1452) [inline experiment-rollout.js]: it's kinda impossible so this experiment is useless
```suggestion
```

File: experiment-rollout.js [hunk 2]
```diff
@@ -161,7 +181,7 @@ const data = {
         notes: [
             {
                 title: 'Masked Links',
-                text: 'Current rollout does not include **masked links**.'
+                text: 'Currently, rollouts do not include **masked links**.'
```
Comment (Tolga1452) [inline experiment-rollout.js:164]:
```suggestion
                text: 'Currently, rollouts does not include **masked links**.'
```
shouldn't it be "does" ?

**Review by Tolga1452 — DISMISSED** [review]

File: experiment-rollout.js (hunk 2 above)
Comment (Lordcobcob) [reply experiment-rollout.js:164]: Nope, it should be do.

Rollouts does not isn't grammatically correct.

File: experiment-rollout.js [hunk 3]
```diff
@@ -101,6 +101,26 @@ const data = {
...
+            },
+            {
+                type: 3,
+                value: [2, 50],
+                rate: 50,
+                ranges: [[1, 50]]
+            }
+        ]
+    },
+
```
Comment (Tolga1452) [inline experiment-rollout.js]:
```suggestion
```

**Review by Tolga1452 — DISMISSED** [review]

**Review by Tolga1452 — APPROVED** [review]

**Pull request merged**